### PR TITLE
feat: support modifiers proposal

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
ref: https://github.com/jviereck/regjsparser/pull/127

### CODE REVIEW

File: tests/equiv.js [hunk 1]
```diff
@@ -54,6 +54,10 @@ function astNodesAreEquivalent(a, b) {
       var bNames = Object.keys(b).filter(function(name) { return name != "range" && name != "raw"; });
       var aNameCount = aNames.length;
 
+      if(a.type == "quantifier" && b.type == "quantifier" && a.symbol == null && b.symbol == "+"){
+        b.symbol = undefined;
+      }
+
```
Comment (liuxingbaoyu) [inline tests/equiv.js:60]: Here is the bug fix, when I updated regjsparser to 0.8.4 it failed the test.

**Review by JLHwung — APPROVED** [review]

**Review by nicolo-ribaudo — APPROVED** [review]

**Comment by nicolo-ribaudo** [comment]
cc @bnjmnt4n if you have time for a review!

**Comment by nicolo-ribaudo** [comment]
Ping @bnjmnt4n :)

**Comment by bnjmnt4n** [comment]
Thanks for the PR and sorry for the delay! Will push a release asap. @nicolo-ribaudo I can add you to the repo + npm package if you'd like. Thanks as always for the awesome work on babel!

**Pull request merged**